Epoch: 0:
Start training ...
Traceback (most recent call last):
  File "/workspace/emotion_KD250628/MKD/CREMA_D_MKD/MKD/main.py", line 495, in <module>
    main()
  File "/workspace/emotion_KD250628/MKD/CREMA_D_MKD/MKD/main.py", line 397, in main
    batch_loss, batch_loss_a, batch_loss_v, global_step = train_epoch(args, epoch, model, device,
  File "/workspace/emotion_KD250628/MKD/CREMA_D_MKD/MKD/main.py", line 132, in train_epoch
    score_v = sum([softmax(out_v)[i][label[i]] for i in range(out_v.size(0))])
  File "/workspace/emotion_KD250628/MKD/CREMA_D_MKD/MKD/main.py", line 132, in <listcomp>
    score_v = sum([softmax(out_v)[i][label[i]] for i in range(out_v.size(0))])
KeyboardInterrupt
Exception ignored in atexit callback: <function _start_and_connect_service.<locals>.teardown_atexit at 0x7fe587ebdb40>
Traceback (most recent call last):
  File "/opt/conda/lib/python3.10/site-packages/wandb/sdk/lib/service_connection.py", line 94, in teardown_atexit
    conn.teardown(hooks.exit_code)
  File "/opt/conda/lib/python3.10/site-packages/wandb/sdk/lib/service_connection.py", line 226, in teardown
    self._router.join()
  File "/opt/conda/lib/python3.10/site-packages/wandb/sdk/interface/router.py", line 75, in join
    self._thread.join()
  File "/opt/conda/lib/python3.10/threading.py", line 1096, in join
    self._wait_for_tstate_lock()
  File "/opt/conda/lib/python3.10/threading.py", line 1116, in _wait_for_tstate_lock
    if lock.acquire(block, timeout):
KeyboardInterrupt:
